src/breakpoints.js

```diff
@@ -0,0 +1,7 @@
+export const BREAKPOINTS = {
+  min: 360,
+  small: 680,
+  medium: 720,
+  large: 960,
+  max: 1280,
+}
```

Comment:
How do we feel about the addition of this `max` breakpoint? I think it's needed given the size jump between medium and large with a higher value.

Reply:
I like it!